enterprise/internal/database/perms_store_test.go

```diff
@@ -1026,11 +1025,23 @@ func checkRepoPendingPermsTable(
 
 			haveSpecs = append(haveSpecs, spec)
 		}
+		wantSpecs := expects[id]
 
-		have := fmt.Sprintf("%v", haveSpecs)
-		if have != want {
-			return errors.Errorf("intIDs - id %d: want %q but got %q", id, want, have)
+		//Verify Specs are the same, the ordering might not be the same but the elements/length are
```

Comment:
Nit. :stuck_out_tongue: 
```suggestion
		// Verify Specs are the same, the ordering might not be the same but the elements/length are.
```